### PR TITLE
Update JSON.lua

### PR DESCRIPTION
**Pull request by nixxo**
update json module to last availabe version on author's webpage http://regex.info/blog/lua/json (see changelog around line 1600)

### CODE REVIEW

File: modules/JSON.lua [hunk 1]
```diff
@@ -1293,6 +1593,46 @@ return OBJDEF:new()
 --
 -- Version history:
 --
+--   20170927.26   Use option.null in decoding as well. Thanks to Max Sindwani for the bump, and sorry to Oliver Hitz
```
Comment (nixxo) [inline modules/JSON.lua:1596]: changelog

**Comment by vladimir-kotikov** [comment]
@nixxo, wondering if this is just a regular update to keep things synced or it fixes an issue for you?

**Comment by nixxo** [comment]
I opened the json-lua file and noticed the version and author infos (while the others didn't have anything) and out of curiosity I checked if there was something more up to date. So I made this PR.

I had no problem with the old one. Just an update to the last available version of the modul.

**Comment by vladimir-kotikov** [comment]
Gotcha! Sounds good to me!

**Pull request merged**